CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.3.1
+  - Fix: safely coerce the value of `_@timestamp` to don't crash the plugin [#67](https://github.com/logstash-plugins/logstash-input-gelf/pull/67)
```

Comment:
```suggestion
  - Fix: safely coerce the value of `_@timestamp` to avoid crashing the plugin [#67](https://github.com/logstash-plugins/logstash-input-gelf/pull/67)
```